.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: [2.3, 2.4, 2.5, 2.6, 2.7, 3.0]
```

Comment:
Can you please drop 2.3-2.5, given that they EOL-ed a while ago?

Reply:
Great point! I forgot this 😄

Reply:
Done! We can add github actions badges in README in a next PR 👍